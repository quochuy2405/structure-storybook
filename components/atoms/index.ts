export { default as Button } from './Button'
export { default as Select } from './Select'
export { default as InputTag } from './InputTag'
export { default as TextArea } from './TextArea'
export { default as Title } from './Title'
export { default as TextField } from './TextField'
export { default as StatusTag } from './StatusTag'
export { default as DatePicker } from './DatePicker'
export { default as ButtonIcon } from './ButtonIcon'
export { default as Avatar } from './Avatar'
export { default as Badge } from './Badge'
export { default as CheckBox } from './CheckBox'
export { default as Card } from './Card'
export { default as Switch } from './Switch'
export { default as SpeedDial } from './SpeedDial'
export { default as Loading } from './Loading'
