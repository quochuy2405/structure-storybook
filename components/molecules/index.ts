export { default as CheckBox } from './CheckBox'
export { default as Modal } from './Modal'
export { default as SelectForm } from './SelectForm'
export { default as TextFieldForm } from './TextFieldForm'
export { default as TextAreaForm } from './TextAreaForm'
export { default as Toast } from './Toast'
