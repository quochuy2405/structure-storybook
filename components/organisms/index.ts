export { default as Tab } from './Tab'
export { default as Report } from './Report'
export { default as Header } from './Header'
