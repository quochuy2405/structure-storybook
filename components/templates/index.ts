export { ArticleCreatePage, ArticleDetailPage, ArticleEditPage } from './ArticlePage'
export { BlogCreatePage, BlogDetailPage, BlogEditPage } from './BlogPage'
export { default as HomePage } from './HomePage'
export { default as LoginPage } from './LoginPage'
export { default as PredictionPage } from './PredictionPage'
export { default as PricePage } from './PricePage'
