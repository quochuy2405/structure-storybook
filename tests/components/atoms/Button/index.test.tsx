import { Button } from '@/components/atoms'
import { render } from '@testing-library/react'
describe('', () => {
  it('renders', () => {
    const { container } = render(<Button />)
    expect(container).toMatchSnapshot()
  })
})
