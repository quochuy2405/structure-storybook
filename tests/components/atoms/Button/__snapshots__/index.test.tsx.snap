// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` renders 1`] = `
<div>
  <button
    class="Button"
  >
    <div>
      Default
    </div>
  </button>
</div>
`;
